### DISCUSSION TITLE
Dashboard card - unable to retrieve statistics from cluster .....

### DISCUSSION THREAD

**Discussion by stadox** [rackslab/Slurm-web#382] [Q&A] [answered]
Hello,

For some reason the Dashboard card shows some issue.

Other TABS (Jobs, Resources,... ) are working well.

![dashboard](https://github.com/user-attachments/assets/dcec7d5a-493c-434b-b11a-b4acaefc6a9f)

![log](https://github.com/user-attachments/assets/1234b18c-4580-49cf-b919-19130ce3dade)

Thank you very much

Kind regards,
Stanislav

**Comment by rezib** [comment]
What version of Slurm are you using?

**Reply by stadox** [reply]
slurmd version 24.05.3 started

**Reply by rezib** [reply]
What is the output of `curl --silent --unix-socket /run/slurmrestd/slurmrestd.socket http://slurm/slurm/v0.0.39/ping` ?

**Reply by stadox** [reply]
```json
{
  "meta": {
    "plugin": {
      "type": "openapi\/v0.0.39",
      "name": "Slurm OpenAPI v0.0.39",
      "data_parser": "v0.0.39"
    },
    "client": {
      "source": "\/run\/slurmrestd\/slurmrestd.socket->socket:[949881] (fd 8)"
    },
    "Slurm": {
      "version": {
        "major": 24,
        "micro": 3,
        "minor": 5
      },
      "release": "24.05.3"
    }
  },
  "errors": [
  ],
  "warnings": [
  ],
  "pings": [
    {
      "hostname": "model07",
      "pinged": "UP",
      "latency": 822,
      "mode": "primary"
    }
  ]
```

**Reply by rezib** [reply]
This is weird, the agent is complaining about missing `Slurm` key in the `meta` object while it is present. Can capture slurmrestd logs along with the agent logs while reproducing the error?

**Reply by stadox** [reply]
Pardon, maybe I didn't correctly get it.

This is log from journalctl -xeu slurmrestd.service when I was trying to see Dashboard

![slurmrestlog](https://github.com/user-attachments/assets/a1b1bd71-48b3-4117-bdb7-316d8281a908)

any other log please let me know

Thank you

Stanislav

**Reply by rezib** [reply, marked as answer]
It looks like you setup `[slurmrestd]` > `version=0.0.41` in `/etc/slurm-web/agent.ini`. There is a [clear statement](https://docs.rackslab.io/slurm-web/conf/conf/agent.html#_slurmrestd) you should not do this in the documentation… Can you please remove this setting and retry?

**Reply by stadox** [reply]
I do very apologize for the version=0.0.41
I have removed that and now it works well.

This setting was probably written there in some weird mistake.

Thank you for your time and patience

Regards,
Stanislav
